### DISCUSSION TITLE
Date Created and other machine read data: any way to help it understand?

### DISCUSSION THREAD

**Discussion by dirtrobber** [jonaswinkler/paperless-ng#1516] [Support]
Paperless-NG is great and I love it but I wondering if I could improve the accuracy of the date created data. Most of the documents I have are in German and state "Datum 00.00.0000" but I routinely have the issue of it reading the wrong date on the document. Is there a way to have it ignore my birthday as a date? This is probably the most incorrectly date tagged on the documents.

**Comment by TheFehr** [comment]
Check out #1349
